config_defaults_inc.php

```diff
@@ -3048,10 +3048,25 @@
 /**
  * limit reporters. Set to ON if you wish to limit reporters to only viewing
  * bugs that they report.
+ * This feature is deprecated and is replaced with the option 'limit_view_unless_threshold'.
+ * Set this option as OFF to be able to use the new one.
+ * @see $g_limit_view_unless_threshold
  * @global integer $g_limit_reporters
```

Comment:
Add `@deprecated` tag with version number (2.23.0 seems reasonable, I don't think this will be merged before 2.22)